dart/lib/src/protocol/sentry_span.dart

```diff
@@ -26,12 +29,14 @@ class SentrySpan extends ISentrySpan {
     this._context,
     this._hub, {
     bool? sampled,
+    Function()? finishedCallback,
   }) {
     this.sampled = sampled;
+    _finishedCallback = finishedCallback;
   }
 
   @override
-  Future<void> finish({SpanStatus? status}) async {
+  Future<void> finish({SpanStatus? status, Duration? autoFinishAfter}) async {
```

Comment:
can we get the `autoFinishAfter` via ctor as well?

Reply:
Ah, i forgot to remove this, sry!